GeoCAT-comp-examples/matplotlib/calendar_average_example.py

```diff
@@ -0,0 +1,123 @@
+"""
+calendar_average_example.py
+===========================
+This script illustrates the following concepts:
+    - Usage of geocat-comp's `calendar_average <https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.climatologies.calendar_average.html?highlight=CALENDER%20average>`_ function.
+    - Usage of geocat-datafiles for accessing NetCDF files
+    - Creating a figure with stacked subplots
+
+See following GitHub repositories to see further information about the function and how to access
+data:
+    - For GeoCAT functions: https://github.com/NCAR/geocat-comp
+    - For atm.20C.hourly6-1990-1995.nc file: https://github.com/NCAR/geocat-datafiles/tree/main/netcdf_files
+
+Dependencies:
+    - cftime
+    - geocat.comp
+    - geocat.datafiles (Not necessary for figure but used for accessing the data file)
+    - geocat.viz
+    - matplotlib
+    - xarray
```

Comment:
cftime and xarray is automatically installed when you have geocat-comp dependency (from a conda environment setup perspective), so we may want to put this information in parenthesis next to each of these packages here.

Reply:
That makes sense. Should I also remove matplotlib from this list since it is automatically installed with geocat.viz?

Reply:
Maybe instead of removing these packages from the list, just give the information that through which package these packages are auto-installed.